### ISSUE TITLE
User saw our side drawer go black on iPhone 6+

### ISSUE THREAD

**Issue by brandtdaniels** [mutualmobile/MMDrawerController#339]
One of my users saw the side drawer go black on iPhone 6+ (iOS 8.1.3). I can't reproduce and don't have much more information, but wanted to record it here.

**Comment by kcharwood** [comment]
We had lots of discussion around this issue here:

https://github.com/mutualmobile/MMDrawerController/issues/30

I'm assuming you are using the latest version of the library?

**Comment by brandtdaniels** [comment]
I believe we are mostly up to date. If I see the issue again, and the user who experienced it is able to provide more data then I will add to that issue. Thanks.

**Comment by lionel-SF** [comment]
Hello, friends

i meet this and reproduce it.
At LeftSideViewController we set difference url addresses which is a UITableViewController,
At CenterViewController we set a weView to load the url from leftsideVC; so we can klick left side tableView and change the url address we want.
Important: we want the web view to handle the session cookie for url, so we do not need login again after invalid.After my APP enter background more than 30 minutes,   the session cookie is invalid, login again by using TouchID then see the same page, but it failures , and show the TouchID again , and TouchID success then you will login success and find the left side VC black.
More: This only happened at APP install by ipa file style. and we set the session valid time to 1 minute at DEBUG ,we do not see it, so it hard for me to follow debug.
Looking forward, who can help ? or any ideas.
![img_4986](https://cloud.githubusercontent.com/assets/15133909/17992313/bde0ffc2-6b78-11e6-8a54-77f0f9e199d8.PNG)
